pkg/skaffold/runner/runcontext/context.go

```diff
@@ -70,13 +71,17 @@ func GetRunContext(opts config.SkaffoldOptions, cfg latest.Pipeline) (*RunContex
 		insecureRegistries[r] = true
 	}
 
+	// TODO(dgageot): what about debug?
```

Comment:
currently debug does not do sync. Also during debugging not sure how the restart would impact the session - I guess they would reconnect...but my gut feeling is that we can safely ignore this for now for debug.

Reply:
Thanks, I'll remove the comment!